R/batch-chat.R

```diff
@@ -325,6 +326,8 @@ BatchJob <- R6::R6Class(
 
     retrieve = function() {
       self$results <- batch_retrieve(self$provider, self$batch)
+      log_turns(self$provider, self$result_turns())
```

Comment:
Just to make sure I'm on the right page, we only call `batch_retrieve()` when we get the results from the API. If we already have the stored, completed results, we wouldn't hit this code path, correct?

Reply:
Yup!